config/synthetics_test/config.go

```diff
@@ -0,0 +1,14 @@
+package synthetics_test
+
+import (
+	tjconfig "github.com/crossplane/terrajet/pkg/config"
+)
+
+// Configure configures the monitor group
+func Configure(p *tjconfig.Provider) {
+	p.AddResourceConfigurator("datadog_synthetics_test", func(r *tjconfig.Resource) {
+		r.ExternalName = tjconfig.IdentifierFromProvider
+		r.Kind = "SyntheticsTest"
+		r.ShortGroup = "syntheticstest"
```

Comment:
Following
<img width="310" alt="image" src="https://user-images.githubusercontent.com/518532/169997147-da4f80d3-a041-4a48-92db-3afe918dbe08.png">

I would do the following grouping and naming

```suggestion
		r.Kind = "Test"
		r.ShortGroup = "synthetics"
```

Reply:
Hi Yury, Thank you for your inputs. I've updated group name and kind. Based on other examples, I've updated package name to match API group name `synthetics` as well.